student.rb

```diff
@@ -0,0 +1,14 @@
+require relative 'person'
+
```

Comment:
- [OPTIONAL] The correct syntax is `require_relative` and not `require relative`. If you use the latter, your application will crash and you will get this sort of error:

![image](https://github.com/SergioPeralta22/MV-School/assets/77106962/5e213225-0eb4-47a5-99be-1f488091a915)

Kindly use the correct syntax 💻 🖱️